src/components/BestSellersCardList.astro

```diff
@@ -1,41 +1,121 @@
 ---
 import { Image } from 'astro:assets';
-import heirRebonding from '../images/heirRebonding.webp';
-import hurbalSkincare from '../images/hurbalSkincare.webp';
-import eyelashExtension from '../images/eyelashExtension.webp';
-const bestSellersList = [
-  {
-    image: heirRebonding,
-    title: 'Heir Rebonding',
-    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit',
-    price: '250',
-  },
-  {
-    image: hurbalSkincare,
-    title: 'Hurbal Skincare',
-    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit',
-    price: '350',
-  },
-  {
-    image: eyelashExtension,
-    title: 'Eyelash Extension',
-    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit',
-    price: '299',
-  },
+
+
+
+const bestSellersList1 = [
```

Comment:
@AleksandraWojcikk  ogólnie nie powinno sie w kodzie uzywac: 
```
section1 
sectionOne
section_one
section2
sectionTwo
secondSection
``` 
etc 

Dlatego ze bywa tak, że sekcje zmieniają się kolejnoscia a czasem i zawartoscia, Lepiej uzywac jakichś nazw lub np w tym przypadku, jeżeli jest to poprostu zwiazane z kolejnoscią, możesz uzyc tablice tablic jako jeden obiekt :)

Reply:
Dobrze wiedzieć, nie wiedziałam, że się nie numeruje. Pokombinowałam trochę z jednym obiektem z użyciem tabeli w tabeli. Zerknij proszę, czy już jest lepiej. Każdą z sekcji również jakoś odmiennie ponazywałam.